src/SFA.DAS.EmployerAccounts.Web/Orchestrators/EmployerAccountPayeOrchestrator.cs

```diff
@@ -170,7 +170,7 @@ public virtual async Task<OrchestratorResponse<RemovePayeSchemeViewModel>> GetRe
                     });
 
             var payeResponse = await
-                Mediator.SendAsync(new GetPayeSchemeByRefQuery
+                Mediator.SendAsync(new GetPagedEmployerAccountsQuery
```

Comment:
Why has this changed?  Will that even work?

Reply:
No idea how this happened :)